application/src/main/java/com/sanction/thunder/ThunderModule.java

```diff
@@ -13,16 +13,20 @@
 import io.dropwizard.jackson.Jackson;
 
 import java.util.List;
+import java.util.Objects;
 import javax.inject.Singleton;
 
+/**
+ * A Dagger module that provides dependencies at the top level.
```

Comment:
Maybe we can expand on this a little once I release the new comment format criteria. I want detailed descriptions of how each class tied into the rest.